### DISCUSSION TITLE
why is the 2.4 standalone and manage node no alert from suricata

### DISCUSSION THREAD

**Discussion by majungman** [Security-Onion-Solutions/securityonion#12008] [2.4]
### Version

2.4.0

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

other (please provide detail below)

### Hardware Specs

Meets minimum requirements

### CPU

20

### RAM

100000

### Storage for /

1000

### Storage for /nsm

1000

### Network Traffic Collection

span port

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

why is there no suricata alert after i install the 2.4 ,i dont know what i missed out but there is no error , just that no alert , even if i import some pcap supposed to flag some alert but it doesnt 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by majungman** [comment]
when trying to run so-rule-update
<img width="763" alt="log2" src="https://github.com/Security-Onion-Solutions/securityonion/assets/102951771/7af518d0-0009-43b3-84fd-30df0740e6b7">
the suricata log
<img width="867" alt="log" src="https://github.com/Security-Onion-Solutions/securityonion/assets/102951771/551f8bbf-cfd7-44ac-8bda-12ea116b3196">
only have the IDH alerts
<img width="818" alt="log3" src="https://github.com/Security-Onion-Solutions/securityonion/assets/102951771/27a0d39f-0f03-40b3-b770-1c9f0929bd6f">

**Comment by TotieBash** [comment]
The IP address CIDR block of the network you want to monitor, is it in Suricata "HOME_NET"? 

SO > Administration > Configuration > suricata > config > vars > address-groups > HOME_NET

By default the following RFC1918 CIDR blocks are entered:
192.168.0.0/16
10.0.0.0/8
172.16.0.0/12

If your network is inside these CIDR blocks then the setting is correct.  If it is not, then add your CIDR block.

**Reply by majungman** [reply]
but there is not a single alert , i tried ping using kali in the home_net range but no alerts too.

**Comment by dougburks** [comment]
Looking at your first screenshot, it's failing to download the rules. Do you have Internet access? If not, you might consider an airgap installation:
https://docs.securityonion.net/en/2.4/airgap.html

For additional troubleshooting steps, please see the Troubleshooting Alerts section of the documentation:
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts

**Reply by majungman** [reply]
hihi, thanks for the reply,
is it all the rules will be in the /nsm/rules/suricata ? because when i ls there is nothing in it . is that the issues ? 
<img width="343" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/102951771/f7588337-5767-4234-9dca-ce7741ef656d">

**Reply by majungman** [reply]
and how can i import the rules ? or where (soup location ?)   can i copy the files to /nsm/rules/suricata?

**Reply by dougburks** [reply]
If your machine is supposed to have Internet access then you should figure out why it's not reaching the Internet and then it should automatically download the rules for you. 

If your machine is NOT supposed to have Internet access, then you will want to choose the airgap option.
